Dockerfile

```
@@ -0,0 +1,6 @@
+# syntax=docker:dockerfile:1
+FROM python:3.10-slim@sha256:2bac43769ace90ebd3ad83e5392295e25dfc58e58543d3ab326c3330b505283d
+WORKDIR /usr/app
+COPY . .
+RUN pip install -r ./Zhuang/requirements.txt
```

```suggestion
RUN python -m pip install -r ./Zhuang/requirements.txt
```

While this is fine, it is best to get into using the pattern `python -m pip` _everywhere_ as it is the only way to be sure that you're _actually_ using the same `pip` that you think you are.